web/deeamon128/javascript/weather-app/app.js

```diff
@@ -0,0 +1,32 @@
+window.addEventListener("load", () => {
+  let locationTimeZone = document.querySelector('.location-timezone');
+  let temperatureDegree = document.querySelector('.temperature-degree');
+  let temperatureDescription = document.querySelector('.temperature-description');
+
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(position => {
+
+      const api = `https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline/London?unitGroup=uk&key=E9R3A3G5GD6CPRK8S7YZBLCVB`;
```

Comment:
Normally, you would want to extract the URL params into variables.
```
const country = 'uk'
const APIkey = 'E9R3A3G5GD6CPRK8S7YZBLCVB'

const api = `https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline/London?unitGroup=${country}&key=${APIkey}`
```

Reply:
thank you